Documentation/Contributing.md

```diff
@@ -228,6 +229,41 @@ If you are only changing the contents of an existing template, not adding new te
 >
 > This feature flag is intended only for short term test usage, and should not be used as a long term solution.
 
+## Deploying your own gallery
+If you are adding new items to a gallery, or adding new gallery entirely, you can use the feature flag `feature.workbookGalleryRedirect` to redirect the entire workbook gallery to a url that you control.
+
+1. clone the repo, create your local branch, and make your changes locally
```

Comment:
Can you capitalize all the letters at the start of each step? It's kind of bothering me :P